src/Pos/DB/Block/Block.hs

```diff
@@ -0,0 +1,302 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Interface to Blocks DB.
```

Comment:
and c) why we're not storing one block per file (because we want to avoid fragmentation?)